circle.yml

```diff
@@ -42,9 +42,9 @@ jobs:
           command: |
             curl -L https://github.com/mozilla/grcov/releases/latest/download/grcov-linux-x86_64.tar.bz2 | tar jxf -
       - run:
-          name: Pull Rust Nightly
+          name: Pull Rust Stable
           command: |
-            docker pull rustlang/rust:nightly-stretch
+             docker pull circleci/rust:1.40-stretch
```

Comment:
I don't think that `stable` will work -- code coverage requires using a bunch of build flags, in particular `-Zprofile` which aren't stable.

Reply:
Didn't know that.  CircleCI timed out when using nightly, so I thought I'd change it.  Will change it back.